google/cloud/storage/oauth2/credential_constants.h

```diff
@@ -32,24 +32,26 @@ namespace oauth2 {
  */
 enum class JwtSigningAlgorithms { RS256 };
 
-/// The endpoint to create an access token from.
-inline char const* GoogleOAuthRefreshEndpoint() {
-  static constexpr char kEndpoint[] = "https://oauth2.googleapis.com/token";
-  return kEndpoint;
-}
-
 /// The max lifetime in seconds of an access token.
 constexpr std::chrono::seconds GoogleOAuthAccessTokenLifetime() {
   return std::chrono::seconds(3600);
 }
 
-/// The skew in seconds, to be subtracted from a token's expiration time,
-/// used to determine if we should attempt to refresh and get a new access
-/// token. This helps avoid a token potentially expiring mid-request.
-constexpr std::chrono::seconds GoogleOAuthTokenExpirationSlack() {
+/**
+ * The skew in seconds, to be subtracted from a token's expiration time,
```

Comment:
Consider a shorter one-liner description, for example:

```C++
/**
 * Defines the slack to avoid tokens expiration while using them.
```

Also, the units are implied by the return type: `std::chrono::seconds`.

Reply:
Fixed.